### PR TITLE
add StoreServer example

### PR DESCRIPTION
**Pull request by laihaotao**
according to the previous slides, add the StoreServer example

### CODE REVIEW

File: examples/ws/soap/StoreExample/StoreServer/WebContent/WEB-INF/MyStoreServerService/DefaultNamespace/deploy.wsdd.bak [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!-- Use this file to deploy some handlers/chains and services      -->
```
Comment (smokhov) [inline examples/ws/soap/StoreExample/StoreServer/WebContent/WEB-INF/MyStoreServerService/DefaultNamespace/deploy.wsdd.bak]: Why do we need a `.bak` file?

File: examples/ws/soap/StoreExample/StoreServer/src/MyStoreServer.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import java.util.HashMap;
+
+public class MyStoreServer {
+	
+	HashMap<String, Integer> inventory = new HashMap<>();
```
Comment (smokhov) [inline examples/ws/soap/StoreExample/StoreServer/src/MyStoreServer.java]: Naming is off again, but I guess it comes from the sample?

Comment (laihaotao) [reply examples/ws/soap/StoreExample/StoreServer/src/MyStoreServer.java]: it is from the sample, will change it.

File: examples/ws/soap/StoreExample/StoreServer/WebContent/WEB-INF/MyStoreServerService/DefaultNamespace/deploy.wsdd.bak (hunk 1 above)
Comment (laihaotao) [reply examples/ws/soap/StoreExample/StoreServer/WebContent/WEB-INF/MyStoreServerService/DefaultNamespace/deploy.wsdd.bak]: I think that file is automatically generated, no idea what it is.

Comment (smokhov) [reply examples/ws/soap/StoreExample/StoreServer/WebContent/WEB-INF/MyStoreServerService/DefaultNamespace/deploy.wsdd.bak]: Then we should not add it to the repo.

**Pull request merged**